include/utils.h

```diff
@@ -16,6 +16,7 @@ void close_preserve_errno(filedesc fd);
 G_DEFINE_AUTO_CLEANUP_FREE_FUNC(filedesc, close_preserve_errno, -1)
 
 #define R_LOG_DOMAIN_SUBPROCESS "rauc-subprocess"
+#define __maybe_unused __attribute__((__unused__))
```

Comment:
GLib already has a G_GNUC_UNUSED macro, looks like it would be the same.

Reply:
Thanks, I'll see if I can just use that instead.